lib/board.rb

```diff
@@ -61,4 +61,14 @@ def place(ship, coordinates)
   def overlapping_ships?(coordinates)
     coordinates.any? { |coordinate| cells[coordinate].ship }
   end
+
+  def render(expose_ship = false)
+    header = "  1 2 3 4 \n"
+    rows = ("A".."D").map do |row|
+      cells_in_row = (1..4).map { |column| cells["#{row}#{column}"].render(expose_ship) }
```

Comment:
This looks really cool and such a great use of the map method to iterate and create our board! I am looking forward to see how this plays out once we play the game.